packages/builder/src/components/application/Form.tsx

```diff
@@ -55,6 +60,242 @@ enum ValidationStatus {
+      />
+      <p className="text-teal-500 font-medium text-xs">Verified</p>
+    </span>
+  );
+
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-2 border-b-2 pt-2 pb-6">
+      {projectRecipient && (
+        <span className="flex items-center mt-4 gap-1">
+          <BoltIcon className="h-4 w-4 mr-1 opacity-40" />
+          <DetailSummary
+            text={`${projectRecipient}`}
+            testID="project-recipient"
+            sm
+          />
+        </span>
+      )}
+      {projectWebsite && (
+        <span className="flex items-center mt-4 gap-1">
+          <GlobeAltIcon className="h-4 w-4 mr-1 opacity-40" />
+          <a
+            href={projectWebsite}
+            target="_blank"
+            rel="noreferrer"
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary
+              text={`${projectWebsite}`}
+              testID="project-website"
+            />
+          </a>
+        </span>
+      )}
+      {projectTwitter && (
+        <span className="flex items-center mt-4 gap-1">
+          {/* <TwitterIcon className="h-4 w-4 mr-1 opacity-40" /> */}
+          <a
+            href={`https://twitter.com/${projectTwitter}`}
+            target="_blank"
+            rel="noreferrer"
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary
+              text={`@${projectTwitter}`}
+              testID="project-twitter"
+            />
+          </a>
+          {getVerifiableCredentialVerificationResultView("twitter")}
+        </span>
+      )}
+      {projectToRender.createdAt && (
+        <span className="flex items-center mt-4 gap-1">
+          {/* <CalendarIcon className="h-4 w-4 mr-1 opacity-80" /> */}
+          <DetailSummary
+            text={`${formattedDateWithOrdinal}`}
+            testID="project-createdAt"
+          />
+        </span>
+      )}
+      {userGithub && (
+        <span className="flex items-center mt-4 gap-1">
+          {/* <GithubIcon className="h-4 w-4 mr-1 opacity-40" /> */}
+          <a
+            href={`https://github.com/${userGithub}`}
+            target="_blank"
+            rel="noreferrer"
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary text={`${userGithub}`} testID="user-github" />
+          </a>
+        </span>
+      )}
+      {projectGithub && (
+        <span className="flex items-center mt-4 gap-1">
+          {/* <GithubIcon className="h-4 w-4 mr-1 opacity-40" /> */}
+          <a
+            href={`https://github.com/${projectGithub}`}
+            target="_blank"
+            rel="noreferrer"
+            className="text-base font-normal text-black"
+          >
+            <DetailSummary text={`${projectGithub}`} testID="project-github" />
+          </a>
+          {getVerifiableCredentialVerificationResultView("github")}
+        </span>
+      )}
+    </div>
+  );
+}
+
+function FullPreview(props: {
```

Comment:
Once I get the CSS to behave I'll break this down in smaller components, similar to the way it's done in GE.